ui/lib/sync/addon/components/secrets/page/destinations/destination/secrets.hbs

```diff
@@ -3,4 +3,85 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<Secrets::DestinationHeaderTabsToolbar @destination={{@destination}} />
\ No newline at end of file
+<Secrets::DestinationHeader @destination={{@destination}} />
+
+{{#if @associations.meta.filteredTotal}}
+  <div class="has-bottom-margin-s">
+    {{#each @associations as |association index|}}
+      <ListItem as |Item|>
+        <Item.content>
+          <div>
+            <span data-test-association-name={{index}}>
+              {{association.id}}
+            </span>
+            <Hds::Badge @text={{association.syncStatus}} @color="neutral" data-test-association-status={{index}} />
+            <div>
+              <code class="has-text-grey is-size-8" data-test-association-updated={{index}}>
+                last synced on
+                {{date-format association.updatedAt "MMMM do yyyy, h:mm:ss a"}}
+              </code>
+            </div>
+          </div>
+        </Item.content>
+
+        <Item.menu as |Menu|>
+          {{#if (or association.setAssociationPath.isLoading association.removeAssociationPath.isLoading)}}
+            <li class="action">
+              <LoadingDropdownOption />
+            </li>
+          {{else}}
+            <li>
+              <Hds::Button
+                @text="Sync now"
+                class="link"
+                disabled={{not association.canSync}}
+                data-test-association-action="sync"
+                {{on "click" (fn this.update association "set")}}
+              />
+            </li>
+            <li>
+              <LinkToExternal
+                class="has-text-black has-text-weight-semibold"
+                data-test-association-action="edit"
+                @route="secret"
+                @models={{array association.mount association.secretName}}
+              >
+                Edit secret
+              </LinkToExternal>
+            </li>
+            {{#if association.canUnsync}}
+              <li>
+                <Menu.Message
+                  data-test-association-action="unsync"
+                  @id={{concat "unsync-" association.id}}
+                  @triggerText="Unsync"
+                  @title="Are you sure?"
+                  @message="This secret will be unsynced from all destinations."
+                  @onConfirm={{fn this.update association "remove"}}
+                />
+              </li>
+            {{/if}}
+          {{/if}}
+        </Item.menu>
+      </ListItem>
+    {{/each}}
+    <Hds::Pagination::Numbered
+      @currentPage={{@associations.meta.currentPage}}
+      @currentPageSize={{@associations.meta.pageSize}}
+      @route="secrets.destinations"
+      @showSizeSelector={{false}}
+      @totalItems={{@associations.meta.filteredTotal}}
+      @queryFunction={{this.paginationQueryParams}}
+      data-test-pagination
+    />
+  </div>
+{{else}}
+  <EmptyState
+    @title="No synced secrets yet"
```

Comment:
This reads incorrectly to me (and I confirmed Grammarly feels the same). It should be `No secrets synced yet`. Maybe this phrasing it used in other places, but I'm fairly certain the noun and verb are backwards.

Reply:
I just took it directly from the designs. Not sure I want to make the call on changing that. I can leave a comment in Figma.

Reply:
I left a comment and it turns out that marketing prepared an empty state message with an image and different copy altogether so this will be getting changed.